app/src/main/AndroidManifest.xml

```diff
@@ -9,15 +8,16 @@
         android:fullBackupContent="@xml/backup_rules"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
+        android:launchMode="singleInstance"
```

Comment:
should be done below for the main activity and not for the application 🙌

Reply:
oops well seen